### PR TITLE
[DM-1989] [DOC] Add availability and prerequisites of LWM2M

### PR DESCRIPTION
**Pull request by JacekGonera**
Add information about agent prerequsites

### CODE REVIEW

File: content/protocol-integration/lwm2m-bundle/overview.md [hunk 1]
```diff
@@ -18,6 +18,10 @@ Lightweight M2M (LWM2M) is a traffic and resource-optimized protocol to remotely
 You can connect any device supporting LWM2M 1.1 or LWM2M 1.0 to {{< product-c8y-iot >}} without programming. {{< product-c8y-iot >}} expects the device and its capabilities (such as firmware update) to be compliant to the LWM2M specification. The device must support the UDP binding of the LWM2M standard.
 {{< /c8y-admon-info >}}
 
+{{< c8y-admon-important >}}
```
Comment (BeateRixen) [inline content/protocol-integration/lwm2m-bundle/overview.md:21]: @rssag I suggest to better create a requirement admonition here?

Comment (rssag) [reply content/protocol-integration/lwm2m-bundle/overview.md:21]: in this exact place?

File: content/protocol-integration/lwm2m-bundle/overview.md [hunk 2]
```diff
@@ -18,6 +18,10 @@ Lightweight M2M (LWM2M) is a traffic and resource-optimized protocol to remotely
...
+In order to use LWM2M, you must be subscribed to the LWM2M-agent application. If the LWM2M-agent is not available in your tenant, please contact [product support](/welcome/contacting-support/).
```
Comment (rssag) [inline content/protocol-integration/lwm2m-bundle/overview.md:22]:
```suggestion
In order to use LWM2M, you must be subscribed to the LWM2M-agent application. If the LWM2M-agent is not available in your tenant, [contact product support](/welcome/contacting-support/).
```

File: content/protocol-integration/lwm2m-bundle/overview.md (hunk 1 above)
Comment (rssag) [inline content/protocol-integration/lwm2m-bundle/overview.md:21]:
```suggestion
{{< c8y-admon-req >}}
```

File: content/protocol-integration/lwm2m-bundle/overview.md [hunk 3]
```diff
@@ -18,6 +18,10 @@ Lightweight M2M (LWM2M) is a traffic and resource-optimized protocol to remotely
...
+{{< /c8y-admon-important >}}
```
Comment (rssag) [inline content/protocol-integration/lwm2m-bundle/overview.md:23]:
```suggestion
{{< /c8y-admon-req >}}
```

**Review by rssag — CHANGES_REQUESTED** [review]

File: content/protocol-integration/lwm2m-bundle/overview.md (hunk 1 above)
Comment (JacekGonera) [reply content/protocol-integration/lwm2m-bundle/overview.md]: Commited the suggestions.

**Review by rssag — APPROVED** [review]

**Review by dominikschmitz — APPROVED** [review]

**Pull request merged**